### PR TITLE
Update contributing.md (pre-MVP phase)

### LINKED ISSUE #439: Update CONTRIBUTING.md - pre-MVP scope
Still minimal content. Focus on expectations.

* Highlight: creating issues/discussions, contribute to docs, contribute bugfixes. We cannot accept all.
* Link to [Contributing](https://github.com/mozilla-ai/lumigator/blob/19b9adefe4a78c42b784461d92b92838b9662564/CONTRIBUTING.md)
* Link to [Docs](https://mozilla-ai.github.io/lumigator/)

### PR DESCRIPTION
**Pull request by ividal**
## What's changing
Closes #439 . Pre-MVP phase (i.e. no feature PRs).

## I already...

- [NA] added some tests for any new functionality
- [X] updated the documentation
- [NA] checked if a (backend) DB migration step was required and included it if required

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,21 @@
 # Contributing
 
-Lumigator is still in early stages of development and as such will have 
-large variance between even PATCH versions in terms of feature development. 
-While we're in this phase, we're accepting PRs for clarification in documentation. 
+Lumigator is still in early stages of development and as such will have
+large variance between even PATCH versions in terms of feature development.
 
-## Project structure
+We do want encourage both suggestions and contributions from the community in the near future.
```
Comment (dpoulopoulos) [inline CONTRIBUTING.md:6]: Suggestion: Put these two sentences in the same paragraph:

```suggestion
We do want encourage both suggestions and contributions from the community in the near future. Being very transparent, please understand that we cannot accept all PRs, so please first open the discussion in the issues.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,9 +1,21 @@
...
 
-The lumigator project contains several packages: the `lumigator/python/mzai/backend` package that powers the lumigator server functionality, the `lumigator/python/mzai/schemas` package containing the formal schemas for communication with the server through the REST API, and the `lumigator/python/mzai/sdk` package abstracting the REST API for Python applications. Each package holds its own `pyproject.toml` definition.
\ No newline at end of file
+Being very transparent, please understand that we cannot accept all PRs, so please first open the discussion in the issues.
+
+As long as this text is here, we are able to accept:
```
Comment (dpoulopoulos) [inline CONTRIBUTING.md:10]: Suggestion: Be more specific; which "text"?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,9 +1,21 @@
...
+
+* PRs for **bugfixes** and clarification in **documentation** only;
```
Comment (dpoulopoulos) [inline CONTRIBUTING.md:12]:
```suggestion
* PRs for [**bugfixes**](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+) and **documentation** only. Please include as much information as possible;
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,9 +1,21 @@
...
+* [bug reports](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+), gladly: please include as much information as possible;
```
Comment (dpoulopoulos) [inline CONTRIBUTING.md:13]: Remove this line if you accept the previous comment:

```suggestion
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,9 +1,21 @@
...
+* [suggestions](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) on making Lumigator easier to understand and contribute to.
+
+
+# Whare to go
```
Comment (dpoulopoulos) [inline CONTRIBUTING.md:17]:
```suggestion
# Where to go
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,9 +1,21 @@
...
+
+* [Lumigator documentation](https://mozilla-ai.github.io/lumigator/), describing all the components and including usage guides;
```
Comment (dpoulopoulos) [inline CONTRIBUTING.md:19]:
```suggestion
* [Lumigator documentation](https://mozilla-ai.github.io/lumigator/): Describes all the components and includes usage guides;
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,9 +1,21 @@
...
+* [open issues](https://github.com/mozilla-ai/lumigator/issues);
```
Comment (dpoulopoulos) [inline CONTRIBUTING.md:20]:
```suggestion
* [Open issues](https://github.com/mozilla-ai/lumigator/issues): Browse the open issues to check if the topic you want to raise is already being discussed.
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,9 +1,21 @@
...
+* [Contributing](https://github.com/mozilla-ai/lumigator/blob/19b9adefe4a78c42b784461d92b92838b9662564/CONTRIBUTING.md) options.
```
Comment (dpoulopoulos) [inline CONTRIBUTING.md:21]: You are linking to `CONTRIBUTING.md` but the user is already in this file. I suggest the following:

```suggestion
* [Contribute](https://github.com/mozilla-ai/lumigator/contribute): Explore the list of "good first issues" to start contributing.
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,9 +1,21 @@
...
-While we're in this phase, we're accepting PRs for clarification in documentation. 
+Lumigator is still in early stages of development and as such will have
+large variance between even PATCH versions in terms of feature development.
 
-## Project structure
+We do want encourage both suggestions and contributions from the community in the near future.
 
-The lumigator project contains several packages: the `lumigator/python/mzai/backend` package that powers the lumigator server functionality, the `lumigator/python/mzai/schemas` package containing the formal schemas for communication with the server through the REST API, and the `lumigator/python/mzai/sdk` package abstracting the REST API for Python applications. Each package holds its own `pyproject.toml` definition.
\ No newline at end of file
+Being very transparent, please understand that we cannot accept all PRs, so please first open the discussion in the issues.
```
Comment (dpoulopoulos) [inline CONTRIBUTING.md:8]: Suggestion: Remove this line if you accept the previous comment.

```suggestion
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,9 +1,21 @@
...
+
+As long as this text is here, we are able to accept:
+
+* PRs for **bugfixes** and clarification in **documentation** only;
+* [bug reports](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+), gladly: please include as much information as possible;
+* [suggestions](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) on making Lumigator easier to understand and contribute to.
```
Comment (dpoulopoulos) [inline CONTRIBUTING.md:14]:
```suggestion
* [Suggestions](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) on making Lumigator easier to understand and contribute to.
```

**Comment by ividal** [comment]
⚠️ Rebased on main.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -1,9 +1,18 @@
 # Contributing
 
-Lumigator is still in early stages of development and as such will have 
-large variance between even PATCH versions in terms of feature development. 
-While we're in this phase, we're accepting PRs for clarification in documentation. 
+Lumigator is still in early stages of development and as such will have
+large variance between even PATCH versions in terms of feature development.
 
-## Project structure
+We want to encourage both suggestions and contributions from the community in the near future. Before submitting a PR, please open a discussion in a related issue.
```
Comment (davidmanzanoai) [inline CONTRIBUTING.md:6]: I would remove "in the near future". I don't think we have a crowd looking at our repo so if someone comes and wants to contribute we will find the way to make that happen (if the contribution makes sense)

**Review by aittalam — APPROVED** [review]

**Comment by aittalam** [comment]
Thanks Irina! LGTM after the rebase

File: CONTRIBUTING.md (hunk 11 above)
Comment (ividal) [reply CONTRIBUTING.md:6]: Removed!

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -1,9 +1,18 @@
...
+
+* PRs for [**bugfixes**](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+) and **documentation** only. Please include as much information as possible;
+* [Suggestions](https://github.com/mozilla-ai/lumigator/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) on making Lumigator easier to understand and contribute to.
+
+
+# Where to go
+
+* [Lumigator documentation](https://mozilla-ai.github.io/lumigator/): Describes all the components and includes usage guides.
+* [Open issues](https://github.com/mozilla-ai/lumigator/issues): Browse the open issues to check if the topic you want to raise is already being discussed.
+* [Contributing](https://github.com/mozilla-ai/lumigator/blob/19b9adefe4a78c42b784461d92b92838b9662564/CONTRIBUTING.md) options.
```
Comment (dpoulopoulos) [inline CONTRIBUTING.md:18]: @ividal are we changing this to:

```suggestion
[Contribute](https://github.com/mozilla-ai/lumigator/contribute): Explore the list of "good first issues" to start contributing.
```

**Pull request merged**